PivotCalibration/Logic/vtkSlicerPivotCalibrationLogic.cxx

```diff
@@ -42,6 +42,13 @@
 
 
 static const double PARALLEL_ANGLE_THRESHOLD_DEGREES = 20.0;
+// Note: If the needle orientation protocol changes, only the definitions of shaftAxis and secondaryAxes need to be changed
+// Define the shaft axis and the secondary shaft axis
+// Current needle orientation protocol dictates: shaft axis -z, orthogonal axis +x
+// If StylusX is parallel to ShaftAxis then: shaft axis -z, orthogonal axis +y
+static const double SHAFT_AXIS[ 3 ] = { 0, 0, -1 };
+static const double ORTHOGONAL_AXIS[ 3 ] = { 1, 0, 0 };
+static const double BACKUP_AXIS[ 3 ] = { 1, 0, 0 };
```

Comment:
Good catch. It should be { 0, 1, 0 }, as described in the comment above it. I'll update the pull request.